openkfw/TruBudget
# Rework editing of permissions

# Changes
The user now has to submit the changes that were made to the permissions of a project, subproject or workflow item. 

Closes #245 

# How to test
Change the permissions for e.g. a project, click submit and the permission should have changed. Try the same and hit cancel instead and the permissions should be the same as before opening the dialog.
